gpii/node_modules/testing/src/CloudBasedOAuth2TestsUtils.js

```diff
@@ -151,22 +132,25 @@ gpii.test.cloudBased.oauth2.buildDisruptedFixture = function (testDef, commonRec
         testDef.name += " - " + disruption.name;
     }
     testDef.gradeNames = testDef.gradeNames.concat(fluid.makeArray(options.testCaseGradeNames));
-    testDef.sequence = gpii.test.elementsBetween(fluid.getGlobalValue(options.sequenceName));
 
     testDef.expectedStatusCode = disruption.expectedStatusCode || options.expectedStatusCode;
-    testDef = gpii.test.pouch.addPouchFixturesToTestDef(testDef);
 
-    fluid.log("oauth2.buildDisruptedFixture issuing sequence" + JSON.stringify(testDef.sequence, null, 2));
     var changes = fluid.makeArray(options.changes).concat(fluid.makeArray(disruption.changes));
     fluid.set(testDef, [options.recordName, "changes"], changes);
 
     return testDef;
 };
 
+// Marker grade for "disruptions", whose unique variations are superimposed on a "common" test definition.
```

Comment:
Should pick a better name for this since "disruptions" will not exist after you have abolished all the array splicing nonsense